src/LibMtpSharpStandardMacOS/LibMtpSharpStandardMacOS.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>

```

Comment:
I think there is something wrong with this project. 
I've cloned this directly and trying to build. 

Files in this location `../../lib/MacOS/output/lib/` do not exist. 
In general this quite strange location for content files. I think that build process should be adjusted or documented. 

There should be some task which will be building dependencies. This files should be compiled every time, or this is one time hit and could be placed inside lib folder? 

What are preconditions needed to build this project?
Maybe it will be better to use old good .nuspec file?

Reply:
Added description how to use this library in .md file